tests/fixtures/example_transformations/add_content_properties/multiple/config.yaml

```diff
@@ -0,0 +1,13 @@
+add_content_properties:
+- class_name: File
+  target_content:
+    object_path: ""
+    property_name: "additional_information"
+  required: false
```

Comment:
Would be good to also test with required=true.

Reply:
That is the default, I've added that information to the config description and also added a case where it is explicitly set to `True`.

Reply:
Is there a reasons why the default is True? It's a bit unecpected.